docs/source/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+/* 24px margin from readthedocs theme */
+div.nblast {
+    margin-bottom: 19px !important;  /* padding has already 5px */
```

Comment:
Thanks @Carreau! I added your suggestion in 9c56f4577db0b9c823d4f34c3350f982ef835368.